jrpcfs/config.go

```diff
@@ -299,5 +309,16 @@ func Down() (err error) {
 	err = nil
 	jsonRpcServerDown()
 	ioServerDown()
+
+	globals.Lock()
+	for _, listner := range globals.listners {
+		listner.Close()
+	}
+
+	for elm := globals.connections.Front(); elm != nil; elm = elm.Next() {
```

Comment:
Ah... and per my earlier comment about this connections list being updated as each goroutine exits... this looping would need to be careful to lock/unlock around accesses...

The way you've done it... you are just sending close's... to a lock/unlock around the loop is fine.

Reply:
yes, that way the list is consistent till all close() calls are done and then the list becomes empty as each connections close.